app/views/nominations/already_associated_with_another_school.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :title, "Fail: induction tutor already nominated" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-header-l">
+      That email address is already associated with another school
+    </h1>
+
+    <h2 class="govuk-heading-m">
+      What to do next
+    </h2>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Ask your colleagues about it</li>
+      <li>Contact DfE on <%= render MailToSupportComponent.new %></li>
```

Comment:
The govuk copy style guide says that titles before a list should end in a colon and list items should start with a lowercase letter

Reply:
<img width="554" alt="Screenshot 2021-04-01 at 13 55 29" src="https://user-images.githubusercontent.com/78035269/113296976-021b0c00-92f2-11eb-92b2-c098444dfa0a.png">

Reply:
That's what the Miro Board says and it is one source of truth that I use

Reply:
The copy for this is all wrong. The source of truth for copy is the copy doc, not miro